robocorp-code/package.json

```diff
@@ -556,7 +559,12 @@
             },
             {
                 "command": "robocorp.inspector",
-                "title": "Open Robocorp Inspector (Experimental)",
+                "title": "Open Inspector Applications",
```

Comment:
I'd drop the 'Applications' -word, because the category is 'robocorp' just having 'Open Inspector' is enough.